metadatastore/odm_templates.py

```diff
@@ -23,7 +23,7 @@ class BeamlineConfig(DynamicDocument):
     config_params = DictField(required=False, unique=False)
     uid = StringField(required=True, unique=True)
     time = FloatField(required=True)
-    meta = {'indexes': ['-_id'], 'db_alias': ALIAS}
+    meta = {'indexes': ['-_id', '-uid'], 'db_alias': ALIAS}
```

Comment:
does not matter for the uids because they are random.  Objectids are prefixed by a time stamp so they are monotonically increasing.